### ISSUE TITLE
SkeletonLoader missing role="status" and aria-label for screen reader users

### ISSUE THREAD

**Issue by aschung212** [aschung212/Lift#272]
No description

**Comment by aschung212** [comment]
Completed by overnight automation on 2026-04-07.

Added role=

[View commit](https://github.com/aschung212/Lift/commit/2ab3cefdc6bb3f1fae1b75fdd20639d9607e92e9)
